src/components/LinePlot/LinePlot.tsx

```diff
@@ -95,8 +95,12 @@ export const LinePlot: React.FC<IProps> = ({ data }) => {
         }) => (
           <div className={classes.tooltipRoot}>
             <Typography className={classes.tooltipDate}>{formatDate(x as number)}</Typography>
-            <Typography className={classes.tooltipValue}>${(y as number).toFixed(2)}</Typography>
-            <FiberManualRecordIcon className={classes.tooltipPoint}/>
+            <Typography className={classes.tooltipValue}>
+              {data.id !== 'userCount'
```

Comment:
hmm, maybe numbers used here in toFixed function could be got from props instaed of being hardcoded?